package.json

```diff
@@ -28,9 +28,10 @@
     "rendering"
   ],
   "dependencies": {
-    "gulp-util": "^3.0.3",
+    "handlebars": ">=3.0.0",
+    "plugin-error": "^0.1.2",
     "through2": "^0.6.3",
-    "handlebars": ">=3.0.0"
+    "vinyl": "^2.1.0"
```

Comment:
You should move vinyl to the devDependencies, since it's only used in the tests.

Reply:
Fair enough, forgot about that 👍